src/dapp/README.md

```diff
@@ -298,9 +298,13 @@ variables](../hevm/README.md#environment-variables).
 | `DAPP_BUILD_OPTIMIZE_RUNS` | `200`                      | Set the optimizer runs                                                                                                                             |
 | `DAPP_TEST_MATCH`          | n/a                        | Only run test methods matching a regex                                                                                                             |
 | `DAPP_TEST_VERBOSITY`      | `0`                        | Sets how much detail `dapp test` logs. Verbosity `1` shows traces for failing tests, `2` shows logs for all tests, `3` shows traces for all tests  |
-| `DAPP_TEST_FUZZ_RUNS`      | `200`                      | How many iterations to use for each property test in your project                                                                                  |
-| `DAPP_TEST_SMTTIMEOUT`     | `600000`                   | Timeout passed to the smt solver for symbolic tests (in ms, and per smt query)                                                                     |
 | `DAPP_TEST_FFI `           | `0`                        | Allow use of the ffi cheatcode in tests (`0` or `1`)                                                                                               |
+| `DAPP_TEST_FUZZ_RUNS`      | `200`                      | How many iterations to use for each property test in your project                                                                                  |
+| `DAPP_TEST_SMTTIMEOUT`     | `30000`                    | Timeout passed to the smt solver for symbolic tests (in ms, and per smt query)                                                                     |
+| `DAPP_TEST_MAX_ITERATIONS` | n/a                        | The number of times hevm will revisit a particular branching point when symbolically executing                                                     |
```

Comment:
i couldn't find the default anywhere. do you know @d-xo?

Reply:
There is no limit by default (i.e. we will attempt to explore the full execution tree by default).